servlet/security-clientcert/pom.xml

```diff
@@ -60,6 +60,25 @@
                 </plugins>
             </build>
         </profile>
+        
+        <profile>
+            <!-- A profile to be used when running against non-payara servers -->
+            <!-- Excludes payara specific tests in the module -->
+            <id>noPayara</id>
```

Comment:
Not a fan of this in its current format.
Would it be possible to simply have this activate based on whether or not any of the payara profiles have been activated?

Reply:
I will look into how that is doable without me spamming includes everywhere

Reply:
I believe there is a vendor specific section this could be placed. Wouldn't need to add any config then